CHANGES

```diff
@@ -1,3 +1,53 @@
+Changes with njs 0.8.5                                       25 Jun 2024
+
+    nginx modules:
+
+    *) Change: r.variables.var, r.requestText, r.responseText,
+       s.variables.var, and the "data" argument of the s.on() callback
+       with "upload" or "download" event types will now convert bytes
+       invalid in UTF-8 encoding into the replacement character.  When
+       working with binary data, use r.rawVariables.var, r.requestBuffer,
+       r.responseBuffer, s.rawVariables.var, and the "upstream" or
+       "downstream" event type for s.on() instead.
+
+    *) Feature: added timeout argument for shared dictionary methods
+       add(), set() and incr().
+
+    *) Improvement: improved checking for duplicate js_set variables.
```

Comment:
this change also appears in 0.8.4

Reply:
This is a fix on top of similar change in 0.8.4.
I  will rephrase it to make it a fix.